Sources/Changeset.swift

```diff
@@ -76,7 +83,28 @@ public struct Changeset<T: Collection> where T.Iterator.Element: Equatable, T.In
 	  - returns: An array of `Edit` elements.
 	*/
 	public static func edits(from s: T, to t: T) -> [Edit<T.Iterator.Element>] {
-		
+		return edits(from: s, to: t) { $0 == $1 }
```

Comment:
You can just do `edits(from: s, to: t, comparator: ==)`. No need to create an extra closure.